MobileCenter/MobileCenter/Internals/Storage/MSFileUtil.m

```diff
@@ -69,7 +69,7 @@ + (BOOL)deleteFile:(MSFile *)file {
     MSLogVerbose([MSMobileCenter logTag], @"File %@: has been successfully deleted", file.fileURL);
     return YES;
   } else {
-    MSLogError([MSMobileCenter logTag], @"Error deleting file %@: %@", file.fileURL, error.localizedDescription);
+    MSLogError([MSMobileCenter logTag], @"Couldn't delete a file \"%@\": %@", file.fileURL, error.localizedDescription);
```

Comment:
@jaelim-ms Thing is, 'file' here is not any file since we specify it right after. That's why I felt removing the 'a'. Whatever we chose we should make it consistent with other formulations of the same kind in this PR (`Couldn't create ...`, `Couldn't load ...`). @TroubleMakerBen what's your feeling in this?